### ISSUE TITLE
Slow integral on 2D Ultraspherical space

### ISSUE THREAD

**Issue by michakraus** [JuliaApproximation/ApproxFun.jl#516]
The first run of an integral on a 2D ultraspherical space seems to be extremely slow:

```
> using ApproxFun

> U = Ultraspherical(1, 0..1)^2;

> Q = DefiniteIntegral(U[1]) ⊗ DefiniteIntegral(U[2]);

> f = Fun((x,y) -> exp(-x*cos(y)),U);

> @time Q*f
227.574802 seconds (665.16 M allocations: 33.004 GiB, 8.02% gc time)

> @time Q*f
0.063531 seconds (226.20 k allocations: 6.888 MiB)
```

Is this a known issue or somehow expected?

**Comment by dlfivefifty** [comment]
This should be compile time. The issue is that too many variables in operators are templated even though in this case I no longer believe it makes things faster for this case.

**Comment by dlfivefifty** [comment]
Also, there's some compile time bugs in Julia depending on which packages are included. 

I think a change in the newly tagged BandedMatrices triggered this bug, which has been worked around on the development branch but not tagged yet.

**Comment by michakraus** [comment]
Yes, it's compile time, but it's still a bit excessive. Many thanks for the hint, though. Pinning BandedMatrices to v0.2.3 solved that problem for the meantime (pulling master did not):
```
> @time Q*f
  9.630592 seconds (9.91 M allocations: 471.316 MiB, 3.95% gc time)

> @time Q*f
  0.055551 seconds (226.20 k allocations: 6.888 MiB)
```
